pkg/config/kube/conversion.go

```diff
@@ -33,6 +33,17 @@ import (
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	wellKnownPorts = map[int32]protocol.Instance{
+		25:    protocol.TCP,   // SMTP
+		53:    protocol.TCP,   // DNS. Default TCP if not specified.
```

Comment:
The map is used when kube protocol is TCP. If DNS use UDP, it will not query this table. 
When kube protocol is set to TCP and kube conversion cannot detect the application protocol from port name, we will query this table in order not to use protocol sniffing for well know ports.

Reply:
e.b. kube-dns doesn't follow istio naming convention, the new code will use protocol sniffing for it without this map